pkgs/os-specific/linux/nftables/python.nix

```diff
@@ -0,0 +1,26 @@
+{ lib
+, buildPythonPackage
+, setuptools
+, nftables
+}:
+
+buildPythonPackage {
+  pname = "nftables";
+  inherit (nftables) version src meta;
```

Comment:
You can't just inherit `meta` because you'll get `meta.mainProgram`, which isn't correct for the Python package. Instead I did

```nix
  meta = {
    inherit (nftables.meta) description homepage license platforms maintainers;
  };
```